data/human/human missions.txt

```diff
@@ -4965,3 +4965,185 @@ mission "Riley"
+			`	The only exception to the bustle and activity are a young woman and man, both dressed very formally, who are seemingly lost and having no luck approaching anybody. You watch they drift from one person to another, each time being sent off without an answer. Finally, they come towards your direction.`
+			`	Their faces show clearly that they are on their last legs of hope. It is only with reluctance that the woman says, "Excuse me, are you a spaceship captain?"`
+			choice
+				`	"Yes, my name's Captain <last>."`
+				`	"Depends on what you are asking."`
+					goto suspicious
+			`	Her face brightens up with a smile, and she continues, "That's good to hear! My brother and I were searching for someone who could give us a ride to New China, we received notice that our father's last will is going to be published there. Do you think you could bring us there? We will pay you for your service, of course."`
+				goto helpornot
+			label suspicious
+			`    She laughs, "There's no need to be suspicious, my brother and I were just searching for someone who could give us a ride to New China. We received notice that our father's last will is going to be published there. Do you think you could bring us there? We will pay you for your service, of course.`
+			label helpornot
+			choice
+				`	"Of course I can."`
+						goto agreed
+				`	"I'm sorry, but I've got more urgent things to do. You'll need to find someone else."`
+			`	Her smile fades away as you say that, but they both simply nod and walk away out the gates of the spaceport, with both heads looking down.`
+				decline
+			label agreed
+			`    "Fantastic! My name is Sophie, and this is my younger brother," she says, pointing at the man to her side, who goes to shake your hand. "Hi, I'm Liam. We have been looking for a ride for some time now, we've already started worrying that we wouldn't get there in time! You'll have to bring us there quickly."`
+			choice
+				`	"Don't worry, you'll be surprised how fast my ship flies. We'll get there in no time."`
+					goto notime
+				`	"Yes, a lot of captains are quite busy these days."`
+			`	"It seems that way," Sophie agrees. "Everyone feels more tense or uncertain, and there's always more to do and less time to do it."`
+				goto ship
+			label notime
+			`	Liam's voice barely hides his excitement at finding an available captain. "Well, let's see that ship of yours then, and show us if it's as fast as you say it is!"`
+			label ship
+			`	You lead them to your ship, and help them with their bags as they board.`
+				accept
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		log "A last will" `Dropped of two siblings on New China to be told their father's last will. They might need another ride.`
```

Comment:
```suggestion
		log `Helped a pair of siblings named Sophie and Liam by bringing them to New China so they can hear their father's last will.`
```
This is more in line with other logs. Also, this happens: 
![Screenshot_20221219_112202](https://user-images.githubusercontent.com/115441627/208322488-50e3b08e-5eaa-41da-97ff-7ba395575355.png)

Reply:
This should not happen